app/views/content/blog/going-back-and-giving-back.md

```diff
@@ -16,8 +16,6 @@ tags:
   - life as a teacher
 ---
 
-$stories-sandra$
-
 For me, the teachers who stick in my mind are the ones that gave me their time and effort, and really prepared me for my future. I chose to train as a teacher so that I could do the same and give something back.
```

Comment:
This looks a bit weird to me now, just having this bit before a h2 with the rest of the content underneath. Get rid of the h2?

Reply:
Yeah totally agree, it's gone!